src/main/java/com/minecolonies/coremod/commands/generalcommands/RandomTeleportCommand.java

```diff
@@ -19,6 +22,7 @@
 
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Map'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)